tests/unit/comment.test.ts

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2023 Intel Corporation.  All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ */
+
+import 'mocha';
+import { expect } from 'chai';
+import {
+    BlockNode,
+    FunctionDeclareNode,
+    FunctionOwnKind,
+    NativeSignature,
+} from '../../src/semantics/semantics_nodes.js';
+import { FunctionType, WASM } from '../../src/semantics/value_types.js';
+import { builtinTypes } from '../../src/semantics/builtin.js';
+import { Import } from '../../src/utils.js';
+import { FunctionalFuncs } from '../../src/backend/binaryen/utils.js';
+import binaryen from 'binaryen';
+import { arrayBufferTypeInfo } from '../../src/backend/binaryen/glue/packType.js';
+
+describe('testParseNativeSignature', function () {
+    it('ARRAYBUFFER_TO_I32', function () {
+        const func = new FunctionDeclareNode(
+            'funcA',
+            FunctionOwnKind.DEFAULT,
+            new FunctionType(-1, WASM.I32, [
+                builtinTypes.get('ArrayBuffer')!,
+                WASM.I32,
+            ]),
+            new BlockNode([]),
+        );
+        const signatureComment: NativeSignature = {
```

Comment:
Is it possible to test parsing from text?

Reply:
I can test `src/utils: parseComment` function by parsing from text, but when testing `parseNativeSignature` function, the provided `FunctonDeclareNode` is needed.

Reply:
I think we can add test for `parseComment`, the input is a series of valid and invalid string and we will check whether the output object is expected, or in case of invalid input if it can return error